bindings/csharp/Ckzg.Bindings/Ckzg.cs

```diff
@@ -102,6 +102,7 @@ public static unsafe void BlobToKzgCommitment(Span<byte> commitment, ReadOnlySpa
         ThrowOnUninitializedTrustedSetup(ckzgSetup);
         ThrowOnInvalidLength(proof, nameof(proof), BytesPerProof);
         ThrowOnInvalidLength(blob, nameof(blob), BytesPerBlob);
+        ThrowOnInvalidLength(commitment, nameof(commitment), BytesPerCommitment);
```

Comment:
Worth pointing out, this check was missing from the C# bindings.

Looks like the new reference tests caught it.